docs/debug-auth-setup.md

```
@@ -0,0 +1,242 @@
+Azure.
+
+```bash
+keytool -list -v \
+  -keystore debug.keystore \
+  -alias androiddebugkey \
+  -storepass android
+```
+
+The output includes lines like:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)